libs/application/templates/parental-leave/src/fields/Review/ReadOnlyReview.tsx

```diff
@@ -0,0 +1,337 @@
+import React, { FC, useState } from 'react'
```

Comment:
I wonder if we could share some structure between the Review and ReadOnlyReview. If we change the order of one field inside one of the two screens, we would need to update the other, and we might forget we would lead to missing fields. 

The common ground I see between the two, is the getValueViaPath where the id of each field is defined. We could probably create an object where we list the object/keys we need and then we render easy in review or read only mode. Another solution, would be to create a common component for both use cath and use the state machine to determine if we are in review or edit mode. Might be the easier. It would end up something like:

```
{edit ? (
<Input
  id="otherParentName"
  name="otherParentName"
  label={formatMessage(m.otherParentName)}
  ref={register}
/>
) : (
<Text>
  {
    getValueViaPath(
      application.answers,
      'otherParentName',
    ) as string[]
  }
</Text>
)}
```

Reply:
Great point, I was wondering how to solve it.
The latest commit has it merged into one `Review` file with the `editable` prop deciding which part to show.